### ISSUE TITLE
Unsupervised Methods & Metrics

### ISSUE THREAD

**Issue by matciotola** [Huises/Awesome-Pansharpening#4]
I've found this repo very interesting. If you are interested, I suggest to include these papers:

- In the section "Unsupervised Deep Learning Methods"

[Pansharpening by convolutional neural networks in the full resolution framework](https://ieeexplore.ieee.org/abstract/document/9745494)
[Fast Full-Resolution Target-Adaptive CNN-Based Pansharpening Framework](https://www.mdpi.com/2072-4292/15/2/319)

- In the section "Perfomance-Assessment"?

[Full-resolution quality assessment for pansharpening](https://www.mdpi.com/2072-4292/14/8/1808)

Thank you

**Comment by Lihui-Chen** [comment]
Yeah. These papers are interesting and contribute a lot to the pansahrpening.

**Comment by matciotola** [comment]
Thank you so much!

**Comment by matciotola** [comment]
For all these papers there are also codes:

- [Pansharpening by convolutional neural networks in the full resolution framework](https://github.com/matciotola/Z-PNN)
- [Fast Full-Resolution Target-Adaptive CNN-Based Pansharpening Framework](https://github.com/matciotola/fast-z-pnn)
- [Full-resolution quality assessment for pansharpening](https://github.com/matciotola/fr-pansh-eval-tool)

I hope that they can be useful! 
Thank you!

**Comment by jf2980** [comment]
Hi , I want to talk about the IERN project. Could you see the commit I left and give me some reponse?? 
thanks
